ts/parts/RangeSelector.ts

```diff
@@ -1268,22 +1280,24 @@ class RangeSelector {
             input: Highcharts.RangeSelectorInputElement,
             label,
             dateBox,
-            inputGroup = this.inputGroup;
+            inputGroup = this.inputGroup,
+            defaultInputDateParser = this.defaultInputDateParser;
 
         /**
          * @private
          */
         function updateExtremes(): void {
-            var inputValue = input.value,
-                value: (number|undefined) =
-                    (options.inputDateParser || Date.parse)(inputValue as any),
+            var inputValue = input.value as any,
+                value: (number|Highcharts.RangeSelectorParseCallbackFunction|undefined),
                 chartAxis = chart.xAxis[0],
                 dataAxis = chart.scroller && chart.scroller.xAxis ?
                     chart.scroller.xAxis :
                     chartAxis,
                 dataMin = dataAxis.dataMin,
                 dataMax = dataAxis.dataMax;
 
+            value = (options.inputDateParser || defaultInputDateParser)(inputValue, chart.time.useUTC);
```

Comment:
Good! Now, since we added new argument to this function call (`useUTC`), we also need to update documentation  for `options.inputDateParser` and it's TS definition.